LabApiUtilities/src/test/com/microsoft/identity/labapi/utilities/client/LabClientTest.java

```diff
@@ -24,15 +24,19 @@
 
 import com.microsoft.identity.labapi.utilities.TestBuildConfig;
 import com.microsoft.identity.labapi.utilities.authentication.LabApiAuthenticationClient;
+import com.microsoft.identity.labapi.utilities.constants.ProtectionPolicy;
 import com.microsoft.identity.labapi.utilities.constants.TempUserType;
 import com.microsoft.identity.labapi.utilities.constants.UserType;
 import com.microsoft.identity.labapi.utilities.exception.LabApiException;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class LabClientTest {
 
+    final long timeout = 35000;
```

Comment:
Add javadoc/comments to explain what this timeout represents

Reply:
Sure thing

Reply:
This has been removed